command/sidechain/helper.go

```diff
@@ -60,9 +60,8 @@ func GetAccountFromDir(accountDir string) (*wallet.Account, error) {
 // GetValidatorInfo queries CustomSupernetManager, StakeManager and RewardPool smart contracts
 // to retrieve validator info for given address
 func GetValidatorInfo(validatorAddr ethgo.Address, supernetManager, stakeManager types.Address,
```

Comment:
Like the `GetValidatorInfo` function on `rootHelper`, we don't need `supernetManager`, nor `stakeManager` addresses here. Since all validator info is now on child chain, you can remove the `GetValidatorInfo` function from `rootHelper` and just keep this one.